omnibus/config/software/system-probe.rb

```diff
@@ -13,4 +13,15 @@
   if ENV.has_key?('SYSTEM_PROBE_BIN') and not ENV['SYSTEM_PROBE_BIN'].empty?
     copy ENV['SYSTEM_PROBE_BIN'], "#{install_dir}/embedded/bin/system-probe"
   end
+
+  mkdir "#{install_dir}/embedded/share/system-probe/ebpf"
+
+  copy 'pkg/ebpf/c/COPYING', "#{install_dir}/embedded/share/system-probe/ebpf/"
+  copy 'pkg/ebpf/bytecode/tracer-ebpf.o', "#{install_dir}/embedded/share/system-probe/ebpf/"
+  copy 'pkg/ebpf/bytecode/tracer-ebpf-debug.o', "#{install_dir}/embedded/share/system-probe/ebpf/"
+  copy 'pkg/ebpf/c/oom-kill-kern.c', "#{install_dir}/embedded/share/system-probe/ebpf/"
+  copy 'pkg/ebpf/c/tcp-queue-length-kern.c', "#{install_dir}/embedded/share/system-probe/ebpf/"
+
+  copy 'pkg/security/ebpf/c/runtime-security.o', "#{install_dir}/embedded/share/system-probe/ebpf/"
+  copy 'pkg/security/ebpf/c/runtime-security-syscall-wrapper.o', "#{install_dir}/embedded/share/system-probe/ebpf/"
```

Comment:
Do we need to chmod these so they're owned by root ?

Reply:
I don't know much about omnibus but it seems that `chown -R dd-agent:dd-agent /opt/datadog-agent` is done in `omnibus/package-scripts/agent/postinst`

Reply:
Would it be better for them to be owned by root? We can do it by changing the `postinst` script as @lebauce suggested, but I'm not sure we need that.